webapp-django/crashstats/crashstats/views.py

```diff
@@ -1425,6 +1425,18 @@ def handle_middleware_404(crash_id, error_code):
         ]
     context['raw_keys'].sort(key=unicode.lower)
 
+    context['install_time'] = None
```

Comment:
Yeah, that's what I was thinking we were already doing, actually. I'm all in favor of it!